packages/babel-preset-react-app/create.js

```diff
@@ -27,6 +17,7 @@ module.exports = function(api, opts, env) {
   var isEnvProduction = env === 'production';
   var isEnvTest = env === 'test';
   var isFlowEnabled = validateBoolOption('flow', opts.flow, true);
+  var areHelpersEnabled = validateBoolOption('helpers', opts.helpers, true);
```

Comment:
Wait, shouldn't default be false?

Reply:
This is the non-dependency version.

Reply:
Ah it's for app code. Hmm. 

Would it break these instructions?

https://reactjs.org/docs/add-react-to-a-website.html#run-jsx-preprocessor

If so I think it should still be off by default.

Reply:
See https://github.com/facebook/create-react-app/pull/5093/files#diff-e4eb38a3161bed144100754a3e97763dR26

Reply:
Right but I don't want this to break non-commonjs usage.

Reply:
Or maybe we should have a separate entry point for "standalone" usage.